e2e-test/interfaces/org.astarte-platform.rust.e2etest.ForUpdateDeviceDatastream.json

```
@@ -0,0 +1,23 @@
+{
+  "interface_name": "org.astarte-platform.rust.e2etest.ForUpdateDeviceDatastream",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "mappings": [
```

```suggestion
  "description": "Test datastream interface.",
  "doc": "Test interface used to test datastream.",
  "mappings": [
```

similar to the ForUpdateDeviceProperty interface